src/cares_wrap.cc

```diff
@@ -296,6 +300,121 @@ Local<Array> HostentToNames(Environment* env, struct hostent* host) {
   return scope.Escape(names);
 }
 
+/* copies a hostent structure*, returns 0 on success, -1 on error */
+/* this function refers to OpenSIPS */
```

Comment:
(See also https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#developers-certificate-of-origin)